test/asm/long-rpn-expression.asm

```diff
@@ -0,0 +1,1000 @@
+SECTION "sec", ROM0
+
+    db sym+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0+0\
```

Comment:
Doesn't REPT work here? Having this file isn't exactly ideal.

Reply:
I've made it much more compact.